support_matrix/_topics/remote_consoles.md

```diff
@@ -0,0 +1,141 @@
+## Remote Console Support
+
+{{ site.data.product.title_short }} allows you to access the console of virtual machines and instances on the providers it managers using HTML5-based VNC consoles. This section outlines the status of support for remote consoles in {{ site.data.product.title_short }}.
+
+**Note:** For VMware vSphere providers, {{ site.data.product.title_short }} supports the versions of VMRC that VMware supports. However, VMware does not support VMRC for Mac operating systems. See VMware VMRC documentation for more information.
+
+ For all other providers, {{ site.data.product.title_short }} supports HTML5-based VNC consoles. The following have been tested with {{ site.data.product.title_short }} Engine:
+
+- Windows: UltraVNC 1.1.8.8 - http://www.uvnc.com/
+- Linux/Mac: SSVNC 1.0.29 - http://www.karlrunge.com/x11vnc/ssvnc.html. At the time of this writing, this is the only client that supports the VNC repeater configuration.
+
+**Note:** As of 1 September 2015, new versions of Chrome no longer support VMRC consoles. VMRC requires NPAPI plug-in support. Chrome removed support for NPAPI plug-ins in version 45.0.2454, which was released on this date.
+
+### Support on Red Hat Enterprise Linux by Providers and Browsers
+
+The following tables outline the status of support for remote consoles on providers for Red Hat Enterprise Linux 6 and Red Hat Enterprise Linux 7. 
+
+#### Remote Console Support on Red Hat Enterprise Linux 6
+
+| Provider                         | Connection Type | Support | Chrome | Firefox |
+| -------------------------------- | --------------- | ------- | ------ | ------- |
+| VMware vSphere 6.0               | VNC             | Yes     | No     | Yes     |
+|                                  | VMRC            |         | No     | Yes     |
+|                                  | WebMKS          |         | No     | Yes     |
+| VMware vSphere 6.5               | VMRC            | Yes     | No     | Yes     |
+|                                  | WebMKS          |         | No     | Yes     |
```

Comment:
Minor, but not seeing vSphere 6.7 in these lists.  Should be the same support as 6.5

Reply:
Should be addressed by commit "Consolidated Infra Providers into a single table"